skranger/ensemble/ranger_forest_classifier.py

```diff
@@ -295,3 +297,28 @@ def predict_log_proba(self, X):
         """
         proba = self.predict_proba(X)
         return np.log(proba)
+
+    def get_importance_pvalues(self):
+        """Return p-values for variable importance using the fast method from Janitza et al. (2016)
+        """
+
```

Comment:
Since this requires training, let's add this check to the beginning of the method: https://scikit-learn.org/stable/modules/generated/sklearn.utils.validation.check_is_fitted.html